Gemfile

```diff
@@ -53,11 +62,9 @@ gem 'devise'
 gem 'newrelic_rpm'
 gem 'skylight'
 
-group :assets do
-  gem 'coffee-rails'
-  gem 'sass-rails'
-  gem 'uglifier'
-end
+gem 'coffee-rails'
```

Comment:
we can get rid of this as soon as we fix up `./app/assets/javascripts/ckeditor/config.js.coffee`

Reply:
Done